web/src/components/Metrics.tsx

```diff
@@ -24,20 +24,7 @@ function validatedPostProcessor(
 
 export function Metrics() {
   const [searchParams, setSearchParams] = useSearchParams();
-  const [store, setStore] = useState<MetricsStore>(new MetricsStore());
-
-  useEffect(() => {
-    async function loadMetrics() {
-      try {
-        const newStore = await fetchMetrics();
-        setStore(newStore);
-      } catch (error) {
-        console.log("Error loading metrics", error);
```

Comment:
Let's add an `error?: string` field to MetricsStore and Manifest so that the web page can display an error rather than just being stuck in a loading state.

Reply:
And please make the pages display the errors rather than content if manifest.error is set